db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
```

Comment:
Needs to be regenerated: `rake db:drop; rm db/schema.rb; rake db:create db:migrate db:seed db:schema:dump` to fix this.. :+1: